File: Client-Project-SBM-main/Client-Project-main/Client-Project/Client.Application/Features/AdditionalEntity/Dtos/AdditionalEntityDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Client.Application.Features.AdditionalEntity.Dtos
{ 
    public class AdditionalEntityDto
    {
        public int R_id { get; set; }
        public string R_type { get; set; }
        public decimal R_amount { get; set; }
        public int R_quantity { get; set; }
        public DateTime R_date { get; set; }
        public int R_companyId { get; set; }
        public int R_subContractorId { get; set; }
        public string R_subContractorName { get; set; }
    }

}

File: Client-Project-SBM-main/Client-Project-main/Client-Project/Client.Frontend/src/app/components/additional-entity/additional-entity-dto.ts
export interface AdditionalEntityGetDto {
    r_id: number;
    r_type: string;
    r_amount: number;
    r_quantity: number;
    r_date: number;
    r_companyId: number;
    r_subContractorName: string;
    r_subContractorId: number;
}

export interface AdditionalEntityCreateDto {
    type: string;
    amount: number;
    quantity: number;
    date: number;
    companyId: number;
    subContractorId: number;
    createdBy: number;
}

export interface AdditionalEntityUpdateDto {
    id: number;
    type: string;
    amount: number;
    quantity: number;
    date: number;
    companyId: number;
    subContractorId: number;
    updatedBy: number;
}
